src/main.c

```diff
@@ -253,6 +280,8 @@ static void open_prefs_dialog(GSimpleAction* simple, GVariant* parameter, gpoint
 		g_key_file_free(kf);
 		// update view
 		week_view_set_day_span(FOCAL_WEEK_VIEW(fm->weekView), fm->prefs.week_start_day, fm->prefs.week_end_day);
+
+		g_signal_emit_by_name(fm->weekView, "date-range-changed");
```

Comment:
Classes should emit their own signals. This should be moved inside `week_view_set_day_span` in the same fashion as used by e.g. [week_view_next](https://github.com/ohwgiles/focal/blob/6ab46231d666e19c8c41eb954337b472a52f32fd/src/week-view.c#L622)